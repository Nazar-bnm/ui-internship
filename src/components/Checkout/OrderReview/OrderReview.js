import React from 'react';
import cx from 'classnames';

import { Button } from '@/shared';

import './OrderReview.scss';

const CN = 'order-review';

class OrderReview extends React.Component {
  render() {
    return (
      <div className={cx(CN)}>
       Please review all the information on this page.
        <br />
       Press the order now button to confirm your purchase.
        <Button customClass={`${CN}__button`}>order now</Button>
      </div>
    );
  }
}

export default OrderReview;
