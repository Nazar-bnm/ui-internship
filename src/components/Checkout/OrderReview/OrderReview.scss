@import '../../../scss/global_styles';
@import '../../../shared/Button/Button.scss';

.order-review {
  @include flex($direction: column);
  padding: 20px;
  font-family: font(primary-light);
  text-transform: uppercase;

  &__button {
    @extend .black-button;
    margin: 10px 0;
    width: 200px;
  }
}