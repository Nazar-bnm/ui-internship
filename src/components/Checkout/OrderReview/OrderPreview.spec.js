import React from 'react';
import { shallow } from 'enzyme';

import OrderReview from './OrderReview';

describe('<OrderPreview />', () => {
  let wrapper = shallow(<OrderReview />);
  
  test('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
