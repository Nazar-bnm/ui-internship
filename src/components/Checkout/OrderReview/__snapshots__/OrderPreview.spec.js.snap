// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrderPreview /> should match snapshot 1`] = `
<div
  className="order-review"
>
  Please review all the information on this page.
  <br />
  Press the order now button to confirm your purchase.
  <Button
    className="order-review__button"
    icon={null}
    onClick={[Function]}
  >
    order now
  </Button>
</div>
`;
